packages/cli/src/executions/executions.service.ts

```diff
@@ -285,66 +260,50 @@ executionsController.get(
 			results: formattedExecutions,
 			estimated,
 		};
-	}),
-);
-
-/**
- * GET /executions/:id
- */
-executionsController.get(
-	'/:id',
-	ResponseHelper.send(
-		async (
-			req: ExecutionRequest.Get,
-		): Promise<IExecutionResponse | IExecutionFlattedResponse | undefined> => {
-			const { id: executionId } = req.params;
+	}
 
-			const sharedWorkflowIds = await getSharedWorkflowIds(req.user);
+	static async getExecution(
+		req: ExecutionRequest.Get,
+		sharedWorkflowIds: number[],
```

Comment:
nit: The `sharedWorkflowIds` obtained from the controller can actually be called inside this function, since the EE controller invokes the EE services, and the free controller invokes the free services, therefore, the correct function would be called because it is overriden.

Reply:
Oh yeah good catch! I may as well make the change over and see when I get back to this!